gtk/src/lib.rs

```diff
@@ -16,71 +16,71 @@
 //!
 //! See also:
 //!
-//! - [Gtk-rs documentation overview](https://gtk-rs.org/docs-src/)
+//! - [gtk-rs documentation overview](https://gtk-rs.org/docs-src/)
```

Comment:
In most places, we use `Gtk-rs` (for example <https://opencollective.com/gtk-rs> and <https://gtk-rs.org>). So for now, I think it'd be better to keep as is until we make a decision about it.

Reply:
Should probably be `gtk-rs` or `GTK-rs`, but let's handle that separately I guess...

Reply:
Agreed (for context: `gtk-rs` refer to the repository whereas `Gtk-rs` currently somewhat refers to the organization. It's very unclear).